### ISSUE TITLE
Traceback when drag and dropping image files from Nautilus to MyPaint

### ISSUE THREAD

**Issue by nekohayo** [mypaint/mypaint#1165]
#### Description of the problem

As the title says ;)

#### Basic system details

MyPaint version: 2.0.1 as packaged in OpenSUSE Tumbleweed, see below
Operating system: OpenSUSE Tumbleweed (rolling distro)
Desktop environment: GNOME 41. Happens both when running as a Wayland session or as a Xorg session.

#### Steps to reproduce

1. Start MyPaint
2. Drag and drop any image file from Nautilus into the canvas or layers area of mypaint

#### Backtraces or error messages

```python
Mypaint version: 2.0.1+gitexport.478280f
System information: Linux-5.14.14-2-default-x86_64-with-glibc2.34
Using: Python 3.8.12, GTK 3.24.30, GdkPixbuf 2.42.6, Cairo 1.16.0, GLib 2.70.1
Traceback (most recent call last):
  File "/usr/lib/mypaint/gui/drawwindow.py", line 290, _drag_data_received_cb(self=<drawwindow.DrawWindow object at 0x7fe0a066fcc0 (MyPaintDrawWindow at 0x561ec5a02500)>, widget=<drawwindow.DrawWindow object at 0x7fe0a066fcc0 (MyPaintDrawWindow at 0x561ec5a02500)>, context=<__gi__.GdkWaylandDragContext object at 0x7fe099c96840 (GdkWaylandDragContext at 0x561ec59a70b0)>, x=681, y=196, data=<Gtk.SelectionData object at 0x7fe099c75f40 (GtkSelectionData at 0x7ffcab3b0140)>, info=1, time=0)
                # working file.
                uri = rawdata.split("\r\n")[0]
                file_path, _h = lib.glib.filename_from_uri(uri)
  variables: {'uri': (None, []), 'rawdata.split': ('local', <built-in method split of bytes object at 0x7fe099c71f80>)}
TypeError: a bytes-like object is required, not 'str'
```

**Comment by rugk** [comment]
Same here:
```python
Mypaint version: 2.0.1+git.478280f.dirty
System information: Linux-5.18.13-200.fc36.x86_64-x86_64-with-glibc2.33
Using: Python 3.9.9, GTK 3.24.34, GdkPixbuf 2.42.8, Cairo 1.16.0, GLib 2.70.5
Traceback (most recent call last):
  File "/app/lib/mypaint/gui/drawwindow.py", line 290, _drag_data_received_cb(self=<drawwindow.DrawWindow object at 0x7f0ce1ca6980 (MyPaintDrawWindow at 0x5558da1de2a0)>, widget=<drawwindow.DrawWindow object at 0x7f0ce1ca6980 (MyPaintDrawWindow at 0x5558da1de2a0)>, context=<GdkX11.X11DragContext object at 0x7f0ce08e7100 (GdkX11DragContext at 0x5558e045e1e0)>, x=1153, y=881, data=<Gtk.SelectionData object at 0x7f0cd893ca00 (GtkSelectionData at 0x7ffd712552b0)>, info=1, time=97092396)
                # working file.
                uri = rawdata.split("\r\n")[0]
                file_path, _h = lib.glib.filename_from_uri(uri)
  variables: {'uri': (None, []), 'rawdata.split': ('local', <built-in method split of bytes object at 0x7f0cd88f3030>)}
TypeError: a bytes-like object is required, not 'str'
```

## system

from Flatpak
```
flatpak info org.mypaint.MyPaint 

MyPaint - Painting program for digital artists

         Kennung: org.mypaint.MyPaint
             Ref: app/org.mypaint.MyPaint/x86_64/stable
     Architektur: x86_64
           Zweig: stable
         Version: 2.0.1
         License: GPL-2.0+
        Ursprung: flathub
        Sammlung: org.flathub.Stable
    Installation: system
     Installiert: 37,7 MB
Laufzeitumgebung: org.gnome.Platform/x86_64/41
             Sdk: org.gnome.Sdk/x86_64/41

          Commit: 03711350039648c00371ff315b461a79f81081e2cacb88dbd27fae6a52e0060d
          Parent: 6341ab5e45897f46825ef25285349a81b262f9169cfc89646bbf8f0fc71e1510
         Subject: Update to GNOME 41 (6103b244)
            Date: 2022-01-02 04:02:33 +0000
```

Wayland
GNOME 42.3

**Comment by fabbrius** [comment]
I'm having the same issue...I just can't import PNG .... any idea how I can fix it?



```python
Mypaint version: 2.0.1+gitexport.478280f
System information: Linux-5.15.0-58-generic-x86_64-with-glibc2.35
Using: Python 3.10.6, GTK 3.24.33, GdkPixbuf 2.42.8, Cairo 1.16.0, GLib 2.71.3
Traceback (most recent call last):
  File "/usr/lib/mypaint/gui/drawwindow.py", line 290, _drag_data_received_cb(self=<drawwindow.DrawWindow object at 0x7f2929dc9680 (MyPaintDrawWindow at 0x55b6d4fdc260)>, widget=<drawwindow.DrawWindow object at 0x7f2929dc9680 (MyPaintDrawWindow at 0x55b6d4fdc260)>, context=<GdkX11.X11DragContext object at 0x7f291c9943c0 (GdkX11DragContext at 0x55b6d5f59ee0)>, x=1593, y=622, data=<Gtk.SelectionData object at 0x7f291c9a7880 (GtkSelectionData at 0x7ffef3f53b40)>, info=1, time=3000278)
                # working file.
                uri = rawdata.split("\r\n")[0]
                file_path, _h = lib.glib.filename_from_uri(uri)
  variables: {'uri': (None, []), 'rawdata.split': ('local', <built-in method split of bytes object at 0x7f2924101fe0>)}
TypeError: a bytes-like object is required, not 'str'
```

**Comment by AesaraB** [comment]
Fixed in 66b2ba98bd953afa73d0d6ac71040b14a4ea266b
